src/libraries/System.Net.Ping/tests/FunctionalTests/PingTest.cs

```diff
@@ -647,6 +647,20 @@ public async Task SendPingAsyncWithHostAndTimeoutAndBufferAndPingOptions_Unix()
                 });
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.OSX)]
```

Comment:
There's no equivalent behavior on other platforms?

Reply:
There is but it's somewhat inconsistent and may report timeout instead. I drafted this only to see what happens on the CI.

Reply:
Basically the size limit can differ between platforms and the error reported can be `TimedOut` (on Unix, non-privileged run) or `PacketTooBig` (macOS, iOS, Windows, privileged Unix). I am not sure how much effort will it take to test all the combinations and cover them by the tests so I started small.